health/server_internal_test.go

```diff
@@ -81,3 +86,65 @@ func (s) TestShutdown(t *testing.T) {
 		t.Fatalf("status for %s is %v, want %v", testService, status, healthpb.HealthCheckResponse_NOT_SERVING)
 	}
 }
+
+// TestList verifies that List() returns the health status of all the services if no. of services are within
+// maxAllowedLimits
+func (s) TestList(t *testing.T) {
+	s := NewServer()
+	s.mu.Lock()
+	// Remove the zero value
+	delete(s.statusMap, "")
```

Comment:
What is this for?

Reply:
To reset the inner map to a zero value, since it defaults to a single entry with a `SERVING` status.

See: https://github.com/grpc/grpc-go/pull/8155/files#diff-12fbd28af25409a1fe9a49ad1a272313e21324834b9518516a8f3a0147e016efR54

Reply:
Can't we skip this, though, since we know that, and it's part of the defined behavior of the health service?  Just add `""` into the expectation.

I'd rather not touch internal state in our tests unless it's necessary (or very helpful).

Reply:
Makes sense to me! Thank you :)